lib/src/functions/meta.dart

```diff
@@ -77,6 +79,15 @@ final local = UnmodifiableListView([
             ? argument
             : SassString(argument.toString(), quotes: false)),
         ListSeparator.comma);
+  }),
+  _function("accepts-content", r"$mixin", (arguments) {
+    var mixin = arguments[0].assertMixin("mixin");
+    return SassBoolean(switch (mixin.callable) {
+      AsyncBuiltInCallable() || BuiltInCallable() => false,
+      UserDefinedCallable(declaration: MixinRule(hasContent: var hasContent)) =>
+        hasContent,
+      _ => throw UnsupportedError("Unknown callable type $mixin.")
+    });
```

Comment:
I've changed my mind on how this should behave with built-in mixins—I think it _should_ look for `allowsContent`, for two reasons:

1. This matches the behavior of user-defined mixins, where if _any_ branch references `@content` the entire mixin is considered to "have content".

2. If in the future we add any built-in mixins that _always_ take a content block, we'll definitely want to check `acceptsContent` here for them, and at that point it would be awkward if we had to add a special case or change the behavior for `meta.apply()`.

Could you also update the spec to check explicitly for built-in mixins that can take content blocks?